src/seeded_rng.rs

```diff
@@ -107,7 +105,6 @@ impl SeededRng {
                 *seed_offset += 1;
                 Rc::new(RefCell::new(SeededRng {
                     seed: seed,
-                    print_seed_on_drop: false,
```

Comment:
But this instance actually shouldn't print its seed, even if it happens to be the first one to be dropped: We only ever want the seed from line 54.